lib.properties

```diff
@@ -1,12 +1,14 @@
-VERSION_NAME=1.4.0
+VERSION_NAME=2.0.0
 VERSION_CODE=1
-GROUP=cz.ackee
-BINTRAY_REPO=gradle-plugin
-BINTRAY_ORGANISATION=ackeecz
+GROUP=io.github.ackeecz
 SITE_URL=https://github.com/AckeeCZ/ackee-gradle-plugin
 GIT_URL=https://github.com/AckeeCZ/ackee-gradle-plugin.git
 POM_DEVELOPER_ID=ackee
 POM_DEVELOPER_NAME=Ackee
 POM_DEVELOPER_EMAIL=info@ackee.cz
 POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
+POM_SCM_CONNECTION=scm:git:github.com/AckeeCz/ackee-gradle-plugin.git
```

Comment:
i dont think it matters but `AckeeCz` => `AckeeCZ`

Reply:
d956d576a63e6440578d5cab79a663f352f02ccc